serverless/pages/search-your-data.asciidoc

```diff
@@ -1,26 +1,33 @@
+// :description: Learn options for searching your data in Elasticsearch Serverless.
 // :keywords: serverless, elasticsearch, search
 
-A search query, or query, is a request for information about data in {es} data streams or indices.
+Searching your data in {es-serverless} works the same way as in other Elasticsearch deployments.
+If you haven't used {es} before, you can learn the basics in the {ref}/elasticsearch-intro.html[core {es} documentation].
 
-You can think of a query as a question, written in a way {es} understands. Depending on your data, you can use a query to get answers to questions like:
+You can use the https://www.elastic.co/docs/api/doc/elasticsearch-serverless[{es-serverless} REST APIs] to search your data using any HTTP client, including the <<elasticsearch-clients,{es} client libraries>>, or directly in the Dev Tools <<devtools-run-api-requests-in-the-console, Console>>.
 
-* What processes on my server take longer than 500 milliseconds to respond?
-* What users on my network ran regsvr32.exe within the last week?
-* What pages on my website contain a specific word or phrase?
+You can also run searches using {kibana-ref}/discover.html[Discover] in your project's UI.
 
-You run search queries using the <<elasticsearch-search-your-data-the-search-api,search API>>. The API supports several query types and search methods:
+[TIP]
+====
+Try our hands-on {ref}/quickstart.html#quickstart-list[quick start tutorials] in the core {es} documentation to get started, or check out our https://github.com/elastic/elasticsearch-labs/tree/main/notebooks#readme[Python notebooks].
+====
 
-**Search for exact values.**
-Use {ref}/term-level-queries.html[term-level queries] to filter numbers, dates, IPs, or strings based on exact values or ranges.
+[discrete]
+[[elasticsearch-search-your-data-query-languages-overview]]
+== Query languages
 
-**Full-text search.**
-Use {ref}/full-text-queries.html[full-text queries] to query {ref}/analysis.html#analysis[unstructured text] and find documents that best match query terms. Use <<elasticsearch-reference-search-with-synonyms,synonyms>> to search for words or phrases that have the same or similar meaning.
+Learn about the various query languages you can use to search your data in the {ref}/search-analyze.html[core {es} documentation].
 
-**Vector search.**
-Store dense vectors in {es} and use <<elasticsearch-knn-search,approximate nearest neighbor (ANN) or k-nearest neighbor (kNN) search>> to find similar vectors.
+[discrete]
+[[elasticsearch-search-your-data-learn-more]]
+== Learn more
 
-You can also use Elastic's natural language processing (NLP) model to encode text as sparse or dense vectors. Then use <<elasticsearch-reference-semantic-search,semantic search>> to find data based on the intent and contextual meaning rather than matching keywords.
+The subpages in this section provide some high-level information about three important search-related topics:
```

Comment:
I know we want to refactor this later, but we should avoid hardcoded #s because they're often missed when pages are edited. 

given that this is not a complete intro to search, consider adding `selected` to better set expectations

```suggestion
The subpages in this section provide some high-level information about selected important search-related topics:
```